fmriprep/workflows/bold/resampling.py

```diff
@@ -92,6 +92,8 @@ def init_bold_surf_wf(mem_gb, output_spaces, medial_surface_nan, name='bold_surf
         workflow.__desc__ = """\
 The BOLD time-series, were resampled to surfaces on the following
 spaces: {out_spaces}.
+*Grayordinates* files [@hcppipelines], which combine surface-sampled
+data and volume-sampled data, were also generated.
```

Comment:
Do we want to make this conditional on `cifti_output`? In which case, it might go better back in `init_func_preproc_wf`, which has access to that variable.

Reply:
Thanks, I've just added it to `init_func_preproc_wf` :+1: